### ISSUE TITLE
Should we merge Denmark and Greenland? 

### ISSUE THREAD

**Issue by RVA2869** [govdirectory/website#368]
### Description

Given recent developments #351, should we merge Denmark and Greenland into the Kingdom of Denmark [(Q756617)](https://www.wikidata.org/wiki/Q756617)? The Kingdom of Denmark consists of Denmark, Greenland and the Faroe Islands.

**Comment by Ainali** [comment]
That is a good question, but I feel we should have some people with good Danish knowledge to make the judgement call. To me, it feels odd if Denmark would be a federal subject of Denmark.
Perhaps it is good to open a discussion on [WikiProject Govdirectory/Denmark](https://www.wikidata.org/wiki/Wikidata_talk:WikiProject_Govdirectory/Denmark) (as we also would like to document all queries there first if this is the case)?

**Comment by RVA2869** [comment]
The Netherlands is also such a case. Kingdom of the Netherlands ([Q29999](https://www.wikidata.org/wiki/Q29999))

- Sint Maarten
- Curaçao
- Aruba
- Netherlands

**Comment by Ainali** [comment]
There is no merging need to be made there, so I would say it is a slightly different case. Also, a hypothetical one, as we don't even have queries for those.

**Comment by RVA2869** [comment]
Yes, no merger is needed, but the Netherlands is a country of the Kingdom of the Netherlands.  Similar to Scotland with the United Kingdom and Denmark, Greenland with the Kingdom of Denmark. So the Netherlands should be part of the Kingdom of the Netherlands.

**Comment by Ainali** [comment]
There is one big difference between, for example, Germany and Denmark, and that is that Denmark doesn't have duplicated levels. There is not both a federal government for the Kingdom of Denmark and another level for "the country". And I think it is the same situation in the Netherlands.  This might be a third special case that we need to think more about, or we'll end up with empty country pages.

**Comment by Ainali** [comment]
I have started to work on sublevels for Germany, for example: [Nordrhein-Westfalen](https://www.wikidata.org/wiki/Wikidata:WikiProject_Govdirectory/Germany/Nordrhein-Westfalen).

**Comment by RVA2869** [comment]
I found an interesting map (related for this issue) 
![Map_of_unitary_and_federal_states](https://github.com/govdirectory/website/assets/144518518/e484356a-f922-4e29-8c8f-e99129f68fe3)
[en:wiki Unitary state](https://en.wikipedia.org/wiki/Unitary_state)
